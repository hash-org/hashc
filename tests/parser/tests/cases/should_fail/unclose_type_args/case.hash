let k<t = do_something();
