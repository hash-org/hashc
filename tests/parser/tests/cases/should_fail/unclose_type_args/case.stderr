error: Failed to parse
 --> $DIR/case.hash:1:3
1 |   k<t := do_something();
  |     ^ here
  = note: Expected an expression, however received a `:`.
